Over 300 test failures for `=dev-python/colorcet-2.0.6`

This is the full [build log](https://ppb.chymera.eu/9bd1dd.log). Most issues seem to be `AssertionError`s.

#### ALL software version info

dev-python/param 1.12.3
dev-python/pyct 0.4.8
dev-lang/python 3.10.9

@jbednar I think @TheChymera works on distributing packages for Gentoo and is reporting that the colorcet test suite does not run correctly on their build system. The errors seems real and not Gentoo specific e.g.:

```
[31m[1m__________________________________________________ test_get_cm[linear_grey_10_95_c0_r-v5] __________________________________________________[0m

k = 'linear_grey_10_95_c0_r', v = <matplotlib.colors.LinearSegmentedColormap object at 0x7f6d98b6a980>

    [37m@pytest[39;49;00m.mark.parametrize([33m'[39;49;00m[33mk,v[39;49;00m[33m'[39;49;00m, [96mlist[39;49;00m(cc.cm.items()))[90m[39;49;00m
    [94mdef[39;49;00m [92mtest_get_cm[39;49;00m(k, v):[90m[39;49;00m
        [94mimport[39;49;00m [04m[96mmatplotlib[39;49;00m[04m[96m.[39;49;00m[04m[96mcm[39;49;00m [94mas[39;49;00m [04m[96mmcm[39;49;00m[90m[39;49;00m
>       [94massert[39;49;00m mcm.get_cmap([33m'[39;49;00m[33mcet_[39;49;00m[33m'[39;49;00m + k) [95mis[39;49;00m v[90m[39;49;00m
[1m[31mE       AssertionError: assert <matplotlib.colors.LinearSegmentedColormap object at 0x7f6d985ee710> is <matplotlib.colors.LinearSegmentedColormap object at 0x7f6d98b6a980>[0m
[1m[31mE        +  where <matplotlib.colors.LinearSegmentedColormap object at 0x7f6d985ee710> = <function _get_cmap at 0x7f6d99b716c0>(('cet_' + 'linear_grey_10_95_c0_r'))[0m
[1m[31mE        +    where <function _get_cmap at 0x7f6d99b716c0> = <module 'matplotlib.cm' from '/usr/lib/python3.10/site-packages/matplotlib/cm.py'>.get_cmap[0m

k          = 'linear_grey_10_95_c0_r'
mcm        = <module 'matplotlib.cm' from '/usr/lib/python3.10/site-packages/matplotlib/cm.py'>
v          = <matplotlib.colors.LinearSegmentedColormap object at 0x7f6d98b6a980>
```

--- a/colorcet/tests/test_matplotlib.py
+++ b/colorcet/tests/test_matplotlib.py
@@ -2,6 +2,7 @@ import sys
 
 import pytest
 import colorcet as cc
+from packaging.version import Version
 
 pytest.importorskip('matplotlib')
 
@@ -46,7 +47,9 @@ def test_matplotlib_default_colormap_plot_kbc():
 
 @pytest.mark.parametrize('k,v', list(cc.cm.items()))
 def test_get_cm(k, v):
-    if sys.version_info < (3, 7):
+    import matplotlib as mpl
+
+    if Version(mpl.__version__) < Version("3.5"):
         import matplotlib.cm as mcm
         assert mcm.get_cmap('cet_' + k) is v
     else:

--- a/colorcet/tests/test_matplotlib.py
+++ b/colorcet/tests/test_matplotlib.py
@@ -1,5 +1,3 @@
-import sys
-
 import pytest
 import colorcet as cc
 from packaging.version import Version

--- a/setup.py
+++ b/setup.py
@@ -41,6 +41,7 @@ tests = [
     'nbsmoke >=0.2.6',
     'pytest >=2.8.5',
     'pytest-cov',
+    'packaging',
 ]
 
 extras_require = {

